it_hardware/demo/website_view.xml

```diff
@@ -0,0 +1,270 @@
+                                    <a href="/website/social/twitter" class="s_social_media_twitter" target="_blank">
+                                        <i class="fa fa-twitter rounded-circle rounded shadow-sm" contenteditable="false">&#8203;</i>
+                                    </a>
+                                    <a href="/website/social/linkedin" class="s_social_media_linkedin" target="_blank">
+                                        <i class="fa fa-linkedin rounded-circle rounded shadow-sm" contenteditable="false">&#8203;</i>
+                                    </a>
+                                </div>
+                                <p class="text-center mb-1">250 Executive Park Blvd, Suite 3400 &#8226; San Francisco CA 94134 &#8226; United States</p>
+                                <ul class="list-inline text-center">
+                                    <li class="list-inline-item mx-3">
+                                        <i class="fa fa-1x fa-fw fa-phone me-2" contenteditable="false">&#8203;</i>
+                                        <span class="o_force_ltr">
+                                            <a href="tel:+1(650)555-0111">+1 (650) 555-0111</a>
+                                        </span>
+                                    </li>
+                                    <li class="list-inline-item mx-3">
+                                        <i class="fa fa-1x fa-fw fa-envelope me-2" contenteditable="false">&#8203;</i>
+                                        <span>
+                                            <a href="mailto:info@yourcompany.example.com">info@yourcompany.example.com</a>
+                                        </span>
+                                    </li>
+                                </ul>
+                            </div>
+                        </section>
+                        <section class="s_text_block pt16 pb16" data-snippet="s_text_block" data-name="Logo" style="background-image: none;">
+                            <div class="container">
+                                <div class="row">
+                                    <div class="col-lg-2 offset-lg-5 o_colored_level">
+                                        <a href="/" class="o_footer_logo logo" data-bs-original-title="">
+                                            <img src="/web/image/it_hardware.ir_attachment_527" alt="" class="img-fluid mx-auto img o_we_custom_image" data-original-id="527" data-original-src="/web/image/it_hardware.ir_attachment_527" data-mimetype="image/webp" data-resize-width="690"/>
+                                        </a>
+                                    </div>
+                                </div>
+                            </div>
+                        </section>
+                    </div>
+                </xpath>
+            </data>
+        </field>
+    </record>
+    <record id="ir_ui_view_2979" model="ir.ui.view">
```

Comment:
Is this view useful? I feel like what is displayed is the standard one

Reply:
yes it is useful as it creates a ticket in the helpdesk